### ISSUE TITLE
Proximity sensor paused by default

### ISSUE THREAD

**Issue by headsvk** [shoya140/GlassLogger#3]
I found out that if the "Wink for picture" gesture is not enabled, the proximity sensor is paused and proxraw is not updated (latest version XE 21.3).

A solution is to enable it after every start from adb shell as root:
`echo "0" > /sys/bus/i2c/devices/4-0035/pause`

**Comment by kkai** [comment]
Yes, either  "wink for picture" or "head detection" need to be enabled for the proximity sensor work. I think an initial calibration is also necessary for the proximity sensor to work.
Thanks for the info about the device state.
